### PR TITLE
Bra 27 homework

### PR DESCRIPTION
**Pull request by dev-kucha**

### CODE REVIEW

File: js/lesson-26/part1.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const vehicle = {
+    manufacturer: `AUDI`,
+    model: `A80`,
+    yearOfManufacture: `2020`,
+    averagSpeed: 50,
+    powerReserve: 600,
+    displayAllAbout() {
+        for (let key in vehicle) {
+            if (typeof vehicle[key] !== 'function') {
+                console.log(`${key} : ${this[key]}`)
```
Comment (roskyba) [inline js/lesson-26/part1.js:12]: Also, you can try to use `console.table`. Would be nicely formatted

File: js/lesson-26/part1.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        }
+    },
+
+    getTimeToTripWithoutRest(dist) {
+        return dist / this.averagSpeed;
+    },
+
+    getTimeToTrip(dist) {
+        timeWithoutRest = this.getTimeToTripWithoutRest(dist);
+        restTime = (timeWithoutRest % 4 !== 0) ? Math.trunc(timeWithoutRest / 4) : Math.trunc(timeWithoutRest / 4) - 1;
```
Comment (roskyba) [inline js/lesson-26/part1.js:24]: Good job and usage of ternary operator!

File: js/lesson-26/part1.js (hunk 1 above)
Comment (dev-kucha) [reply js/lesson-26/part1.js:12]: I tried it and I liked it!
[https://dev-kucha.github.io/bra-27-homework/html/lesson-26/part1.html](https://dev-kucha.github.io/bra-27-homework/html/lesson-26/part1.html)
Thanks for the interesting solution!

File: js/lesson-26/part1.js (hunk 2 above)
Comment (dev-kucha) [reply js/lesson-26/part1.js:24]: Thanks :-)

**Pull request merged**